models/user.rb

```diff
@@ -10,7 +10,7 @@ class User < Base
 
   RESET_WINDOW = 60 * 15 # 15 minutes
 
-  SETTINGS = (6.times.flat_map do |index|
+  SETTINGS = (8.times.flat_map do |index|
```

Comment:
Other part of supporting up to 8 routes.

Reply:
hm this shouldn't be hardcoded, can you pull the size of the colors here?

Reply:
I agree. yzemaze hardcoded it for some reason. I can ask him. Are you okay with models dir files referencing assets dir files?

Reply:
maybe

Reply:
Fixed.